client/client.go

```diff
@@ -221,10 +255,34 @@ func (c *Client) processChannelRequests() {
 			responsePayload = new(agentpb.SetStateResponse)
 
 		case *agentpb.StartActionRequest:
-			panic("TODO")
+			var a actions.Action
+			switch p.Type {
+			case managementpb.ActionType_PT_SUMMARY:
+				processParams := p.Params.(*agentpb.StartActionRequest_ProcessParams_)
+				a = actions.NewShellAction(p.ActionId, c.cfg.Paths.PtSummaryAction, processParams.ProcessParams.Args)
+				c.concurrentActionRunner.Run(a)
+				responsePayload = &agentpb.StartActionResponse{}
+
+			case managementpb.ActionType_PT_MYSQL_SUMMARY:
+				processParams := p.Params.(*agentpb.StartActionRequest_ProcessParams_)
+				a = actions.NewShellAction(p.ActionId, c.cfg.Paths.PtMySQLSummaryAction, processParams.ProcessParams.Args)
+				c.concurrentActionRunner.Run(a)
+				responsePayload = &agentpb.StartActionResponse{}
+
+			case managementpb.ActionType_MYSQL_EXPLAIN:
+				// TODO: Implement explain action.
+				c.l.Errorf("not implemented action EXPLAIN")
+				continue
+
+			case managementpb.ActionType_ACTION_TYPE_INVALID:
+				c.l.Errorf("Unsupported action: %s.", p.Type)
+				continue
+			}
 
+		// Handle Action Cancel request from pmm-managed
```

Comment:
All other cases in this switch are for requests from pmm-managed too. Why single out this one?